packages/react-form-state/src/components/tests/List.test.tsx

```diff
@@ -161,6 +161,7 @@ describe('<FormState.List />', () => {
 
     const {title} = lastCallArgs(renderSpy);
     title.onChange(newTitle);
+    wrapper.forceUpdate();
```

Comment:
Just a question: I see `wrapper.forceUpdate();` used a few times in this PR, is this the only way to do this? Was it due to a change from react1t -> 18?

Reply:
In this particular case @royleejr it might make more sense to do `wrapper.act(() => title.onChange(newTitle))`